modsettings/SnapPoints.json

```diff
@@ -356,5 +356,12 @@
       "y": 1.481,
       "z": -56.165
     }
+  },
+  {
+    "Position": {
+      "x": -21.251,
+      "y": 1.481,
+      "z": 55.589
+    }
```

Comment:
I don't think a snap point is needed. Just add a tag to the created coin (e.g. "SCED_Export") and use that to get the token.